### PR TITLE
OSOE-464: Disabling node reuse during dotnet build and test

### PR DESCRIPTION
**Pull request by Piedone**
OSOE-464

### CODE REVIEW

File: .github/actions/build-dotnet/Build-DotNetSolutionOrProject.ps1 [hunk 1]
```diff
@@ -69,6 +69,12 @@ dotnet build $SolutionOrProject @buildSwitches 2>&1 | ForEach-Object {
     if ($noErrors) { Write-Output "::error file=$file,line=$line,col=$column::$message" }
 }
 
+# With node reuse, dotnet build spawns processes that while speed up build, they can cause dotnest test and other dotnet
```
Comment (dministro) [inline .github/actions/build-dotnet/Build-DotNetSolutionOrProject.ps1:72]:
```suggestion
# With node reuse, dotnet build spawns processes that while speed up build, they can cause dotnet test and other dotnet
```

Comment (Piedone) [reply .github/actions/build-dotnet/Build-DotNetSolutionOrProject.ps1:72]: I always have to pay attention not to make this typo :D. I'll fix this in dev.

Comment (sarahelsaig) [reply .github/actions/build-dotnet/Build-DotNetSolutionOrProject.ps1:72]: Typo? So it wasn't guerilla marketing?! 😱

Comment (Piedone) [reply .github/actions/build-dotnet/Build-DotNetSolutionOrProject.ps1:72]: I'll sneak this change into the .NET SDK.

**Pull request merged**